src/hawkbit-client.c

```diff
@@ -741,14 +741,17 @@ gboolean install_complete_cb(gpointer ptr)
  * feedback and call software_ready_cb() callback on success.
  *
  * @param[in] data Artifact* to process
- * @return NULL is always returned
+ * @return pointer to gboolean being TRUE if download succeeded, FALSE otherwise.
```

Comment:
With the change to `GINT_TO_POINTER()` We should be consistent here, too, and say someting like:

``` suggestion
 * @return gpointer being 1 (TRUE) if download succeeded, 0 (FALSE) otherwise. The return value is meant to be used with GPOINTER_TO_INT() macro only.
```

Reply:
Applied :+1: